### PR TITLE
Add dockerfile and gunicorn server

### PR DESCRIPTION
**Pull request by gidsg**
### Link to ticket
https://trello.com/c/inL5eg30/6-deploy-the-app-so-it-can-be-accessed-for-testing-with-users

### Change description
Adds Dockerfile and setup Gunicorn as production-ready webserver. 
(Having a containerised version of the app makes it easier to deploy on AWS)

- [ ] Unit tests and other appropriate tests added or updated **N/A**
- [x] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
* Follow steps in README to run with Docker
* App should be accessible at http://localhost:8080

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+ENV FLASK_ENV=dev
+ENV GUNICORN_CMD_ARGS="--workers=3"
+WORKDIR /app
+COPY requirements.txt requirements.txt
+RUN python3 -m pip install --upgrade pip && pip install -r requirements.txt
+COPY . .
+
+EXPOSE 8080
+
+CMD ["gunicorn", "wsgi:app", "-c", "run/gunicorn/run.py"]
```
Comment (cyrusdobbs) [inline Dockerfile:11]: Do we want to remove root user privileges before we run the application?

**Review by cyrusdobbs — COMMENTED** [review]
Docker build and run has been tested and works fine

File: run/gunicorn/run.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from fsd_utils.gunicorn.config.devtest import *  # noqa
```
[the remark this answers is not in the document]
Comment (gidsg) [reply run/gunicorn/run.py:1]: @cyrusdobbs This references the shared gunicorn config we use across FSD projects: https://github.com/communitiesuk/funding-service-design-utils/blob/main/fsd_utils/gunicorn/config/devtest.py

[the remark this answers is not in the document]
Comment (cyrusdobbs) [reply run/gunicorn/run.py:1]: got it cheers 👍

**Review by cyrusdobbs — APPROVED** [review]
Looks good 👍

**Pull request merged**